gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,50 @@ def get_possible_database_path():
     return path
 
 
+def create_database_directory():
+    """Creates the standard GRASS GIS directory.
+
+    Creates database directory named grassdata in the standard location 
+    according to the platform.  
+    
+    Returns the new path as a string or None if nothing was found or created.
+    """
+    home = os.path.expanduser('~')   
+    candidates = []       
+   
+    # Candidate for case independent "grassdata" for Linux and macOS
+    if sys.platform.startswith('linux') or sys.platform.startswith('darwin'):
+        candidates.append(os.path.join(home, "grassdata"))   
+    # Candidates for case independent "grassdata" in other dirs (Windows)
+    else:  
+        candidates.append(os.path.join(home, "Documents", "grassdata"))
```

Comment:
Simple `candidate = os.path...` will suffice here in both cases. It is either or and at one point you have only one candidate. This removes the need for the loop over the candidates list below.